src/Tribe/Settings_Tab.php

```diff
@@ -120,19 +144,35 @@ public function __construct( $id, $name, $args = [] ) {
 			'show_save'        => true,
 			'display_callback' => false,
 			'network_admin'    => false,
-			'parent'           => null,
```

Comment:
If you don't leave parent as a default value, we may get tabs that throw fatals for not passing one.